xl/player/player.py

```diff
@@ -187,15 +195,48 @@ def play(self, track, start_at=None, paused=False):
         if track is None:
             self.stop()
         else:
+            self.stop()
```

Comment:
Why is this here? I think this would break gapless playback, and it's going to emit a stop event too? The player isn't actually stopping after all.